tests/test_mixed_simulator_device.py

```diff
@@ -708,26 +708,26 @@ def test_four_qubit_random_circuit(self, shots):
         dev = qml.device("cirq.mixedsimulator", wires=4)
 
         gates = [
-            qml.PauliX,
-            qml.PauliY,
-            qml.PauliZ,
-            qml.S,
-            qml.T,
-            qml.RX,
-            qml.RY,
-            qml.RZ,
-            qml.Hadamard,
-            qml.Rot,
-            qml.CRot,
-            qml.Toffoli,
-            qml.SWAP,
-            qml.CSWAP,
-            qml.U1,
-            qml.U2,
-            qml.U3,
-            qml.CRX,
-            qml.CRY,
-            qml.CRZ,
+            qml.PauliX(wires=0),
+            qml.PauliY(wires=1),
+            qml.PauliZ(wires=2),
+            qml.S(wires=3),
+            qml.T(wires=0),
+            qml.RX(2.3, wires=1),
```

Comment:
How were the gate parameters and the wires chosen? Do they correspond to the previously used gate parameters/wires?

Reply:
No I took what Maria used in the example for integration tests.